openfecli/parameters/plan_network_options.py

```diff
@@ -180,21 +180,14 @@ def load_yaml_planner_options(path: Optional[str], context) -> PlanNetworkOption
 
 
 _yaml_help = """\
-Path to planning settings yaml file
-
-Currently it can contain sections for customising the
-atom mapper and network planning algorithm,
-these are addressed using a `mapper:` or `network:` key in the yaml file.
-The algorithm to be used for these sections is then specified by the `method:` key. 
-For choosing mappers, either the LomapAtomMapper or KartografAtomMapper are allowed choices,
-while for the network planning algorithm either the generate_minimal_spanning_tree or
-generate_minimal_redundant_network options are allowed.
-Finally, a `settings:` key can be given to customise the algorithm,
-with allowable options corresponding to the keyword arguments of the Python API for these algorithms.
-
-For example, this is a valid settings yaml file to specify that
-the Lomap atom mapper should be used forbidding element changes,
-while the generate_minimal_redundant_network function used to plan the network
+Path to a YAML file specifying the atom mapper (`mapper:`) and/or network planning algorithm (`network:`) to use.
+
+Supported atom mapper choices include `LomapAtomMapper` or `KartografAtomMapper`.
+Supported network planning algorithms include `generate_minimal_spanning_tree` or `generate_minimal_redundant_network`.
```

Comment:
Maybe also include `generate_radial_network` as that one might be quite common?

Reply:
I agree. We also just need more thorough documentation for `plan_network_options` in general.